Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -693,5 +693,87 @@ function Rtg_UpdateTemperature( furniture, deltaTime)
     --ModUtils.ULogChannel("Temperature", "Heat change: " .. temperatureChangePerSecond .. " => " .. World.current.temperature.GetTemperature(tile.X, tile.Y))
 end
 
+-- Register the temperature update function
+function Fire_InstallAction( furniture, deltaTime)
+	furniture.EventActions.Register("OnUpdateTemperature", "Fire_UpdateTemperature")
+	World.Current.temperature.RegisterSinkOrSource(furniture)
+end
+
+-- Deregister the temperature update function
+function Fire_UninstallAction( furniture, deltaTime)
+	furniture.EventActions.Deregister("OnUpdateTemperature", "Fire_UpdateTemperature")
+	World.Current.temperature.DeregisterSinkOrSource(furniture)
+end
+
+-- Increase temperature of current tile
+function Fire_UpdateTemperature( furniture, deltaTime)
+	-- Magic numbers for temperature
+	World.Current.temperature.SetTemperature(furniture.Tile.X, furniture.Tile.Y, 700 + furniture.Parameters["strength"].ToFloat() * 100)
```

Comment:
Probably not. I only got to making the actual implementation of the fire "furniture" work again in terms of the new furniture system. I haven't gone into the Lua yet